server/lib/tuist/environment.ex

```diff
@@ -80,6 +80,16 @@ defmodule Tuist.Environment do
     not dev?() or truthy?(System.get_env("TUIST_DEV_USE_REMOTE_STORAGE", "0"))
   end
 
+  def prometheus_enabled? do
```

Comment:
Similar to: https://github.com/tuist/tuist/pull/7967/files#r2254583879

The prometheus server was conflicting and it was prohibiting us from running a `mix` and a phoenix server instance side-by-side